### PR TITLE
Use jit in test_randomness (#198)

### PR DESCRIPTION
**Pull request by kiya00**
<details>
  <summary><b>Before submitting</b></summary>

- [ ] Was this discussed/approved via a Github issue? (no need for typos and docs improvements)
- [ ] Did you read the [contributor guideline](https://github.com/Lightning-AI/pytorch-lightning/blob/main/.github/CONTRIBUTING.md), Pull Request section?
- [ ] Did you make sure to update the docs?
- [ ] Did you write any new necessary tests?

</details>

## What does this PR do?

use `thunder.jit` in `test_randomness.py`

### CODE REVIEW

**Comment by kiya00** [comment]
Hi @crcrpar @t-vi , a small change for the test, could you help to review and merge it?

File: thunder/tests/test_randomness.py [hunk 1]
```diff
@@ -20,7 +20,7 @@ def test_uniform_philox(executor, device: str, dtype: dtypes.dtype):
     def func(shape, dtype, device, rng_seed, rng_offset):
         return ltorch.uniform_philox(shape, device=device, dtype=dtype, seed=rng_seed, offset=rng_offset)
 
-    cf = lc_compile(func, disable_preprocessing=True, executors_list=executor.executors_list())
+    cf = thunder.jit(func, executors_list=executor.executors_list())
```
Comment (kshitij12345) [inline thunder/tests/test_randomness.py:23]:
```suggestion
    cf = thunder.jit(func, executors=executor.executors_list())
```

`executors_list` is only supported for compat.

**Review by kshitij12345 — APPROVED** [review]
LGTM, thanks @kiya00

**Review by crcrpar — APPROVED** [review]

**Review by carmocca — APPROVED** [review]

**Comment by kiya00** [comment]
Hi @t-vi  @carmocca , I think it's ready to merge, `test_randomness.py` is passed, CI failed in some unrelated cases

**Pull request merged**